docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -131,7 +131,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `REACTIONS`: All available reactions. Allow users react with different emoji's.
 - `DEFAULT_SHOW_FULL_NAME`: **false**: Whether the full name of the users should be shown where possible. If the full name isn't set, the username will be used.
 - `SEARCH_REPO_DESCRIPTION`: **true**: Whether to search within description at repository search on explore page.
-- `USE_SERVICE_WORKER`: **true**: Whether to enable a Service Worker to cache frontend assets.
+- `USE_SERVICE_WORKER`: **true**: Whether to enable a Service Worker to cache frontend assets and enable push notifications on supported browsers.
```

Comment:
Don't piggy-back on the service-worker pref, these are two distinct features. Add a new one, like `USE_PUSH_NOTIFICATIONS`.

Reply:
True - I'll pop a note saying the service worker is required as well.